app/react/ConnectionsList/components/ResetSearch.js

```diff
@@ -4,19 +4,21 @@ import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 
 import ShowIf from 'app/App/ShowIf';
-import { t } from 'app/I18N';
+import { Translate } from 'app/I18N';
 import { Icon } from 'UI';
 
 import { resetSearch } from '../actions/actions';
 
-export class ResetSearch extends Component {
+class ResetSearch extends Component {
   render() {
     const { connectionsGroups } = this.props;
     return (
       <ShowIf if={Boolean(connectionsGroups.size)}>
-        <button onClick={this.props.resetSearch} className="btn btn-primary">
-          <Icon icon="sync" />
-          <span className="btn-label">{t('System', 'Reset')}</span>
+        <button type="button" onClick={this.props.resetSearch} className="btn btn-default">
+          <Icon icon="times" />
+          <span className="btn-label">
+            <Translate>Clear Filters</Translate>
```

Comment:
Should there be a `translationKey` specified in the props of the `<Translate />`?